torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -121,11 +139,12 @@ RegisterOperators reg({
         onnx::Reshape,
         [](Node* node) {
           return [=](Stack& stack) {
-            auto shape = pop(stack).toTensor().contiguous();
-            auto input = pop(stack).toTensor();
+            at::Tensor input, shape;
```

Comment:
Why do we have to support running `onnx::Reshape`?

Reply:
It is because we don't have a clean pipeline from <python>-><ATen IR>-><ONNX>. Some stuff defined in overrides does <python>-><ONNX> and so we actually see them even when not doing ONNX. We should fix this but it requires cleaning up how onnx overrides work, forcing them to first generate valid script code.